### PR TITLE
Unify error response format

### LINKED ISSUE #351: Standardise error responses
Most of error responses return as `{ "errors": ["messages"] }` but some of them return as `{ "error": "message" }`.

Make all error responses to return as `{ "errors": ["messages"] }` as it seems to be the original format and it matches with the main use case in Windshaft-cartodb: https://github.com/CartoDB/Windshaft-cartodb/issues/305

### PR DESCRIPTION
**Pull request by rochoa**
Replaces `error=>String` with `errors=>Array` error responses

Closes #351

### CODE REVIEW

**Comment by rochoa** [comment]
cc @zenitraM I don't know if this may affect you, so just in case I prefer to bother you.

**Comment by rochoa** [comment]
@alonsogarciapablo did you get to review this one?

**Comment by rochoa** [comment]
@alonsogarciapablo ping?

File: test/acceptance/attributes.js [hunk 1]
```diff
@@ -162,6 +162,7 @@ describe('attributes', function() {
         },
         function checkPost(err, res) {
           assert.ifError(err);
+            console.log('rochoa', res.statusCode, res.body);
```
Comment (alonsogarciapablo) [inline test/acceptance/attributes.js]: This looks like a deja vu... :smile:

**Comment by alonsogarciapablo** [comment]
Sorry @rochoa, I missed your mentions... Other than my comment: LGTM :+1:

**Pull request merged**